be/src/exprs/agg/factory/aggregate_resolver_minmaxany.cpp

```diff
@@ -32,8 +32,22 @@ void AggregateFuncResolver::register_bitmap() {
             "bitmap_union_int", false, AggregateFactory::MakeBitmapUnionIntAggregateFunction<TYPE_INT>());
     add_aggregate_mapping<TYPE_BIGINT, TYPE_BIGINT, BitmapValue>(
             "bitmap_union_int", false, AggregateFactory::MakeBitmapUnionIntAggregateFunction<TYPE_BIGINT>());
-    add_aggregate_mapping<TYPE_OBJECT, TYPE_OBJECT, BitmapValue>("bitmap_union", false,
-                                                                 AggregateFactory::MakeBitmapUnionAggregateFunction());
+
+    add_aggregate_mapping<TYPE_OBJECT, TYPE_OBJECT, BitmapValue>(
+            "bitmap_union", false, AggregateFactory::MakeBitmapUnionAggregateFunction<TYPE_OBJECT>());
+    add_aggregate_mapping<TYPE_BOOLEAN, TYPE_OBJECT, BitmapValue>(
+            "bitmap_union", false, AggregateFactory::MakeBitmapUnionAggregateFunction<TYPE_BOOLEAN>());
+    add_aggregate_mapping<TYPE_TINYINT, TYPE_OBJECT, BitmapValue>(
+            "bitmap_union", false, AggregateFactory::MakeBitmapUnionAggregateFunction<TYPE_TINYINT>());
+    add_aggregate_mapping<TYPE_SMALLINT, TYPE_OBJECT, BitmapValue>(
+            "bitmap_union", false, AggregateFactory::MakeBitmapUnionAggregateFunction<TYPE_SMALLINT>());
+    add_aggregate_mapping<TYPE_INT, TYPE_OBJECT, BitmapValue>(
+            "bitmap_union", false, AggregateFactory::MakeBitmapUnionAggregateFunction<TYPE_INT>());
+    add_aggregate_mapping<TYPE_BIGINT, TYPE_OBJECT, BitmapValue>(
+            "bitmap_union", false, AggregateFactory::MakeBitmapUnionAggregateFunction<TYPE_BIGINT>());
+    add_aggregate_mapping<TYPE_LARGEINT, TYPE_OBJECT, BitmapValue>(
```

Comment:
I think we couldn't support largeint type

Reply:
The id of some user is largeint..., so i covert it to null if it extend uint64_t::max